src/main.cpp

```diff
@@ -6,100 +6,104 @@
 #include "pb_decode.h"
 #include "urc.pb.h"
 
-IntervalTimer encoderMessageTimer;
-elapsedMillis elapsedTime;
+/* Networking Global Variables */
+static IPAddress g_ip(192, 168, 8, 255);
+static EthernetUDP g_udp;
+static volatile bool g_print_flag = false;
+static const byte g_mac[] = { 0xDE, 0xAD, 0xBE, 0xEF, 0xFE, 0xED };
+static const uint16_t UDP_PORT = 8888;
```

Comment:
Raw globals make me uneasy, even if this is embedded code. You sure about making them standard for the repo?

Reply:
Restructured to make more object oriented.